### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by chrischildresssg**
## Change description

> Update the examples for correctness and to demonstrate EFS volumes/mounts

## Type of change
- [ ] Bug fix (fixes an issue)
- [x] New feature (adds functionality)

## Related issues

> Fix (N/A)

### Development

- [x] Lint rules pass locally
- [x] Application changes have been tested thoroughly
- [x] Automated tests covering modified code pass

### Security

- [x] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [x] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Review by wesleykirklandsg — CHANGES_REQUESTED** [review]
@chrischildresssg This won't work since this repo uses TFDocs.

Also on the version, I'm a personal fan of how Cloud Posse does versions https://github.com/cloudposse/terraform-aws-tfstate-backend#usage

File: README.md [hunk 1]
```diff
@@ -80,71 +86,137 @@ module "ecs_fargate_app" {
...
+    taskdef_execution_role_arn = "arn:aws:iam::123456789876:role/my_taskdef_role"
+    taskdef_task_role_arn      = "arn:aws:iam::123456789876:role/my_taskdef_role"
+    taskdef_network_mode       = "awsvpc"
+    taskdef_requires_compatibilities = [
+      "FARGATE"
+    ]
+    taskdef_cpu    = 2048
+    taskdef_memory = 4096
+
+    taskdef_container_definitions = <<-TASKDEF
```
Comment (mattbarlow-sg) [inline README.md:165]: Nit: Consider using HCL in the clients and `jsonencode` in the module which makes for a cleaner client experience and stronger typing enforcement.

**Review by mattbarlow-sg — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (chrischildresssg) [reply README.md:165]: I did - jsonencode() destroys the "<" and ">" in the "<IMAGE1_NAME>" string, which breaks the deployment functionality (sadface).  Are you aware of a way around this problem?

Comment (mattbarlow-sg) [reply README.md:165]: Maybe this comment helps: https://github.com/hashicorp/terraform/pull/18871#issuecomment-422220699

**Review by wesleykirkland — APPROVED** [review]
LGTM. I see what I missed on my documentation.

File: README.md (hunk 1 above)
Comment (chrischildresssg) [reply README.md:166]: This, indeed, is a *much* better approach and will be used in the next revision (coming soon!).

**Review by mattbarlow-sg — APPROVED** [review]

**Review by wesleykirklandsg — APPROVED** [review]
LGTM

**Pull request merged**